pkg/util/templater/template_functions.go

```diff
@@ -63,9 +64,9 @@ func (r *Templater) templateFuncsMap(tm *template.Template) template.FuncMap {
 
 	}
 
-	funcs["ChannelRecommendedImage"] = func(cloud, k8sVersion string) string {
+	funcs["ChannelRecommendedImage"] = func(cloud, k8sVersion string, architecture architectures.Architecture) string {
```

Comment:
normally changing a template function's signature would be worthy of a release note but this hasnt been in a stable release yet so we should be fine. we do need to update the docs that use this function though https://github.com/kubernetes/kops/blob/master/docs/operations/cluster_template.md

Reply:
Done.